### ISSUE TITLE
training script

### ISSUE THREAD

**Issue by Blubberli** [Blubberli/ambiguous-alpaca#14]
this script will contain the main method that can take a configuration file with metadata, e.g. what kind of classifier should be used and the hyper parameter.

- method to read configuration files (we can have a look at allenNLP library, they use json files for that)
- method that does a training loop for a number of epochs
- method that does logging of loss etc.
- method that does apply early stopping
- method to save and load checkpoints and models
